source/guides/references/configuration.md

```diff
@@ -61,15 +62,15 @@ Option | Default | Description
 ----- | ---- | ----
 `screenshotOnRunFailure` | `true` | Whether Cypress will take a screenshot when a test fails during `cypress run`.
 `screenshotsFolder`     | `cypress/screenshots`     | Path to folder where screenshots will be saved from {% url `cy.screenshot()` screenshot %} command or after a test fails during `cypress run`
-`trashAssetsBeforeRuns` | `true` | Whether Cypress will trash assets within the `screenshotsFolder` and `videosFolder` before tests run with `cypress run`.
+`trashAssetsBeforeRuns` | `true` | Whether Cypress will trash assets within the `downloadsFolder`, `screenshotsFolder`, and `videosFolder` before tests run with `cypress run`.
```

Comment:
feels weird to me that there's no `## Downloads` section with `trashAssetsBeforeRuns` and `downloadsFolder`, to match `## Screenshots` and `## Videos`

Reply:
Good call. I'll add a Downloads section.

Reply:
yah I started writing this before that feature existed.